backend/dataall/base/utils/stack_logs_utils.py

```diff
@@ -0,0 +1,33 @@
+from functools import wraps
+from dataall.base.config import config
+from dataall.base.context import get_context
+
+
```

Comment:
See comment in config.json first!
We can make the utility more generic. At the moment it is very very focused in the stacks and it contains a lot of business info (where anything in `base` should be usable outside of data.all concepts, dataset, shares...)

We could add a utility similar to `dataall.base.feature_toggle_checker.is_feature_enabled`, maybe even in the same file

```
def is_feature_allowed_value(config_property: str, allowed_value: Any):
    def decorator(f):
        fn, fn_decorator = process_func(f)

        def decorated(*args, **kwargs):
            value = config.get_property(config_property)
            if value != allowed_value
                raise Exception(f'Disabled by config {config_property}')
            return fn(*args, **kwargs)

        return fn_decorator(decorated)

    return decorator
```

Reply:
@dlpzx , for now, I have added the decorator function sugggested by you in the same file. Please let me know if you have a preference to keep it in another file